### PR TITLE
Update bartholomew commands

### PR DESCRIPTION
**Pull request by tpmccallum**
Content must go through a pre-merge checklist.

## Pre-Merge Content Checklist

This documentation has been checked to ensure that:

- [x] The `title, `template`, and `date` are all set
- [x] Does this PR have a new menu item (anywhere in `templates/*.hbs` files) that points to a document `.md` that is set to publish in the future? If so please only publish the `.md` and `.hbs` changes in real-time (otherwise there will be a menu item pointing to a `.md` file that does not exist)
- [x] File does not use CRLF, but uses plain LF (hint: use `cat -ve <filename> | grep '^M' | wc -l` and expect 0 as a result) 
- [x] Has passed [`bart check`](https://developer.fermyon.com/bartholomew/quickstart)
- [x] Has run `npm run build-index`
- [x] Has been manually tested by running in Spin/Bartholomew (hint: use `PREVIEW_MODE=1` and run `npm run styles` to update styling)
- [x] Headings are using Title Case
- [x] Code blocks have the programming language set to properly highlight syntax and the proper copy directive
- [x] Have tested with `npm run test` and resolved all errors

### CODE REVIEW

File: templates/spin_sidebar.hbs [hunk 1]
```diff
@@ -94,6 +94,7 @@
         <ul class="menu-list">
             <li><a {{#if (active_content request.spin-path-info "/common/contributing-docs" )}} class="active" {{/if}} href="{{site.info.base_url}}/common/contributing-docs">Contributing to Docs</a></li>
             <li><a {{#if (active_content request.spin-path-info "/spin/contributing-spin" )}} class="active" {{/if}} href="{{site.info.base_url}}/spin/contributing-spin">Contributing to Spin</a></li>
+            <li><a href="https://www.fermyon.com/code-of-conduct" target="_blank">Code of Conduct</a></li>
```
Comment (itowlson) [inline templates/spin_sidebar.hbs:97]: Ditto, drop from TOC

**Review by itowlson — CHANGES_REQUESTED** [review]

File: content/bartholomew/quickstart.md [hunk 2]
```diff
@@ -77,19 +83,34 @@ Creating content is made easy with the Bartholomew Command Line Interface (CLI)
...
+$ cd ~
+# Clone Bartholomew repository
 $ git clone https://github.com/fermyon/bartholomew.git
+# Change into Bartholomew directory
 $ cd bartholomew
+# Build the Bartholomew CLI
 $ make bart
 ```
 
+> NOTE: The procedure, in the example code block above, will create the `bart` executable binary file inside the `~/bartholomew/target/release/` directory. Please ensure that this new binary executable is added to your system path.
```
Comment (itowlson) [inline content/bartholomew/quickstart.md:102]: Another rogue comma 
```suggestion
> NOTE: The procedure in the example code block above will create the `bart` executable binary file inside the `~/bartholomew/target/release/` directory. Please ensure that this new binary executable is added to your system path.
```

(or simplify down to "The above procedure creates...")

**Review by itowlson — APPROVED** [review]
One minor grammatical nit but otherwise LGTM - thanks!

**Comment by vdice** [comment]
A note to mention that this closes https://github.com/fermyon/developer/issues/539.

**Review by vdice — APPROVED** [review]

**Pull request merged**